packages/components/src/List/Toolbar/ColumnChooserButton/ColumnChooser/SelectAllColumnsCheckbox/SelectAllColumnsCheckbox.component.js

```diff
@@ -6,32 +6,43 @@ import { getTheme } from '../../../../../theme';
 
 const theme = getTheme(cssModule);
 
-const getDescription = (checked, t) => {
-	if (checked) {
-		return t('CHECKBOX_DESELECT_ALL_COLUMNS_DESCRIPTION', {
-			defaultValue: 'hide all the columns',
-		});
-	}
-	return t('CHECKBOX_SELECT_ALL_COLUMNS_DESCRIPTION', {
-		defaultValue: 'display all the columns',
-	});
+const getLabels = (checked, t) => {
+	return checked
+		? {
+				label: t('TC_COLUMN_CHOOSER_UNSELECT_ALL', {
+					defaultValue: 'Unselect all',
+				}),
+				description: t('CHECKBOX_DESELECT_ALL_COLUMNS_DESCRIPTION', {
+					defaultValue: 'hide all the columns',
+				}),
+		  }
+		: {
+				label: t('TC_COLUMN_CHOOSER_SELECT_ALL', {
+					defaultValue: 'Select all',
+				}),
+				description: t('CHECKBOX_SELECT_ALL_COLUMNS_DESCRIPTION', {
+					defaultValue: 'display all the columns',
+				}),
+		  };
```

Comment:
```suggestion
	return checked
		? {
				label: t('TC_COLUMN_CHOOSER_UNSELECT_ALL',  'Unselect all'),
				description: t('CHECKBOX_DESELECT_ALL_COLUMNS_DESCRIPTION', 'hide all the columns'),
		  }
		: {
				label: t('TC_COLUMN_CHOOSER_SELECT_ALL', 'Select all'),
				description: t('CHECKBOX_SELECT_ALL_COLUMNS_DESCRIPTION', 'display all the columns'),
		  };
```

Reply:
I hope one day be able to detect this simplification by myself.